index.html

```diff
@@ -11,49 +11,205 @@
+                    </div>
+                </div>
+                <div class="grid-item">
+                    <div class="grid-project-image2"></div>
+                    <div class="project-details">
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>JavaScript</p>
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-button">See this project &nbsp;<i
+                                class="fas fa-arrow-right fa-lg"></i></button>
+                    </div>
+                </div>
+                <div class="grid-item">
+                    <div class="grid-project-image3"></div>
+                    <div class="project-details">
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>JavaScript</p>
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-button">See this project &nbsp;<i
+                                class="fas fa-arrow-right fa-lg"></i></button>
+                    </div>
+                </div>
+                <div class="grid-item">
 
-    </section>
+                    <div class="grid-project-image4"></div>
+                    <div class="project-details">
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>JavaScript</p>
+                            </div>
+                        </div>
```

Comment:
- [ ] The list of project technologies ("JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`